config/custom-environment-variables.json

```diff
@@ -50,6 +50,11 @@
   "pdf-service": {
     "url": "PDF_SERVICE_URL"
   },
+  "startPage": {
```

Comment:
I think a better approach would be to delete both this and the response start page now and redirect to eligibility